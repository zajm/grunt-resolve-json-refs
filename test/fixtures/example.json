{
    "A": {
        "A1": "A1",
        "A2": "A2"
    },
    "B": {
        "B1": "@:E"
    },
    "C": "@:A.A1",
    "D": "@:C",
    "E": "ahoi!",
    "F": "D",
    "CAREFUL_NOW": "@:@:F"
}