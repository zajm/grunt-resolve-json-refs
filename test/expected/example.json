{
    "A": {
        "A1": "A1",
        "A2": "A2"
    },
    "B": {
        "B1": "ahoi!"
    },
    "C": "A1",
    "D": "A1",
    "E": "ahoi!",
    "F": "D",
    "CAREFUL_NOW": "A1"
}